docs/conventions.md

```diff
@@ -33,6 +34,19 @@ Branch names should use lower-kebab-case and be prefixed with the issue number:
 _Note: Branch names (and PR names) do NOT have to match the issue title. Instead, try to be short and concise to focus
 on the actual work done._
 
+## Pull Requests
+
+Pull requests should belong to one of our [issues](https://github.com/digitalfabrik/entitlementcard/issues).
+If you are looking for issues to work on, a good place to start are
+our [good first issues](https://github.com/digitalfabrik/entitlementcard/issues?q=state%3Aopen%20label%3A%22good%20first%20issue%22).
+
+To merge a pull request, it has to meet the following criteria:
+
+- All checks (formatting, linting and unit tests) have to pass.
+- No changes are requested.
+- Two approvals are needed.
+- Labels are set, check [Labels](#labels).
```

Comment:
I thought we wanted to use these labels for issues `backend`, `frontend`, `web`.
do we need them on PRs too?